testing/test_basic.py

```diff
@@ -108,12 +109,13 @@ def test_MAIN():
 def test_blocon_in_hook(testdir):
     testdir.makeconftest("""
         import pytest
+        import pytest_twisted as pt
```

Comment:
I don't like this but without it `pytest_twisted` is detected as an unknown pytest hook...  :|

```
INTERNALERROR> PluginValidationError: unknown hook 'pytest_twisted' in plugin <module 'conftest' from '/tmp/pytest-of-altendky/pytest-199/testdir/test_blocon_in_hook0/conftest.py'>
```